File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    wallpaper = [list(row) for row in wallpaper]
    u = len(wallpaper)
    d = 0
    l = len(wallpaper[0])
    r = 0
    row = u
    col = l
    for i in range(row):
        for j in range(col):
            if (wallpaper[i][j] == '#'):

                if (u > i):
                    u = i
                if (d < i):
                    d = i
                if (l > j):
                    l = j
                if (r < j):
                    r = j
    answer = [u,l, d+1,r+1]
    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    vector<int> answer = {51,51,0,0};
    int row = wallpaper.size();
    int col = wallpaper[0].size();
    
    for (int i = 0; i < row; i++) {
        for (int j = 0; j < col; j++) {
            if (wallpaper[i][j] == '#') {
                answer[0] = answer[0] < i ? answer[0] : i;
                answer[1] = answer[1] < j ? answer[1] : j;
                answer[2] = answer[2] > i + 1 ? answer[2] : i + 1;
                answer[3] = answer[3] > j + 1 ? answer[3] : j + 1;
            }
        }
    }
    return answer;
}